scripts/findChrome.js

```diff
@@ -171,7 +171,8 @@ async function downloadChromium() {
     path: chromeTempPath,
   })
 
-  const revision = pptrCoreJson.puppeteer.chromium_revision
+  const revision = process.env.PUPPETEER_CHROMIUM_REVISION || process.env.npm_config_puppeteer_chromium_revision
```

Comment:
We need more `process.env` to copy. For instance, we need `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` and `PUPPETEER_DOWNLOAD_HOST`.

Reply:
Added `DOWNLOAD_HOST` and all kinds of options to skip download.